binrz/man/rz-ax.1

```diff
@@ -1,100 +1,100 @@
-.Dd Dec 28, 2020
+.Dd Jan 24, 2024
 .Dt RZ_AX 1
 .Sh NAME
 .Nm rz-ax
 .Nd rizin base converter
 .Sh SYNOPSIS
 .Nm rz-ax
-.Op Fl ebBsSvxkKh
+.Op Fl abBdefFhiIkLlNnorsStuvwxC
 .Op [expr] ...
 .Sh DESCRIPTION
 This command is part of the rizin project.
 .Pp
-This command allows you to convert values between positive and negative integer, float, octal, binary and hexadecimal values.
+This command allows you to convert values between positive and negative integers, floats, octal, binary, ternary, and hexadecimal values.
 .Sh OPTIONS
 .Bl -tag -width Fl
+.It Fl l
+Append newline to output (for -E/-D/-r/..)
+.It Fl a
+Show ASCII table
 .It Fl b
-Convert from binary string to character (rz-ax \-b 01000101)
-.It Fl k
-Keep the same base as the input data
+Convert from binary string to character (rz-ax -b 01000101 01110110)
+.It Fl B
+Convert from string to binary (rz-ax -B hello)
+.It Fl d
+Force output as integer (rz-ax -d 3 outputs 3 instead of 0x3)
 .It Fl e
 Swap endian.
+.It Fl f
+Floating point arithmetic (rz-ax -f 6.3+2.1)
 .It Fl F
-Read C strings from stdin and output in hexpairs. Useful to load shellcodes
+Read C strings from stdin and output in hex pairs. Useful to load shellcodes
+.It Fl h
+Show usage help message
 .It Fl i
-Dump stdin to C array in stdout (xxd replacement)
+Dump as C byte array
 .It Fl I
 Convert LONG to/from IP ADDRESS
-.It Fl l
-Append newline to the decoded output for human friendly-ness
-.It Fl K
-Show randomart key asciiart for values or hexpairs
-.It Fl s
-Convert from hex string to character (rz-ax \-s 43 4a 50)
-.It Fl S
-Convert from character to hex string (rz-ax \-S C J P)
+.It Fl k
+Keep the same base as the input data
+.It Fl L
+Convert binary to hexadecimal (rz-ax -L 111111111 outputs 0x1ff)
 .It Fl n
-Show hexpairs from integer value
+Show binary number from integer value
 .It Fl N
-Show hexadecimal C string from integer value
+Show binary C string from integer value
+.It Fl o
+Convert from octal string to char (rz-ax -o \162 \172)
+.It Fl r
+Rizin style output
+.It Fl s
+Convert from hex string to character (rz-ax -s 43 4a 50)
+.It Fl S
+Convert from character to hex string (rz-ax -S < /bin/ls > ls.hex)
+.It Fl t
+Convert timestamp to string (rz-ax -t 1234567890)
 .It Fl u
-Convert given value to human readable units format
+Convert given value to human-readable units format
 .It Fl v
-Show program version
+Show version number
+.It Fl w
+Convert signed word (16-bit) (rz-ax -w 16 0xffff)
 .It Fl x
-Convert a string into a hash
-.It Fl h
-Show usage help message
-.It Fl o
-Convert from octal string to char (rz-ax -o 162 62)
+Hash string
+.It Fl p
+Show position of set bits in the binary representation (rz-ax -p 0xb3)
 .El
 .Sh USAGE
 .Pp
 Force output mode (numeric base)
 .Pp
-  =f    floating point
-  =2    binary
-  =3    ternary
-  =8    octal
-  =10   decimal
-  =16   hexadecimal
+=f floating point
+=2 binary
+=3 ternary
+=8 octal
+=10 decimal
+=16 hexadecimal
 .Pp
```

Comment:
it was better before.

before:
![image](https://github.com/rizinorg/rizin/assets/562321/98e82fac-cc86-4e41-b5cc-3512d3ee1344)

after:
![image](https://github.com/rizinorg/rizin/assets/562321/54d686d7-1f11-4989-811a-f38bee2f5691)